api/ocm/tools/toi/drivers/docker/driver.go

```diff
@@ -469,7 +469,7 @@ func generateTar(files map[string]blobaccess.BlobAccess, uid int) (io.ReadCloser
 			dir := path
 			for dir != "/" {
 				dir = unix_path.Dir(dir)
-				if !have[dir] {
+				if dir != "/" && dir != "." && !have[dir] {
```

Comment:
Isn't `dir != "/"` redundant due to the for-loop on L470?

Reply:
Nevermind `unix_path.Dir(dir)` (or rather `path.Dir()`) would cut `/a` to `/`. Thus, it is not covered by the for-loop